packages/eslint-plugin-nx/migrations.json

```diff
@@ -0,0 +1,13 @@
+{
+  "schematics": {},
+  "packageJsonUpdates": {
+    "13.5.3": {
+      "version": "13.5.2",
+      "packages": {
+        "@swc-node/register": {
+          "version": "1.4.2"
+        }
+      }
+    }
+  }
+}
```

Comment:
@meeroslav Why is `@swc-node/register` added to the users package.json? I'm just curious.

Reply:
It should not. I added the `alwaysAddToPackageJson` flag to false

Reply:
@meeroslav Sorry to bother you again, but why is a migration needed anyways?

Reply:
In case user's repo already uses `@swc-node/register` for whatever reason, we want to make sure they have the at least the version we tested with (`core` version before 1.6 did not support path aliases).

Reply:
@meeroslav Maybe I'm wrong, but when a user's repo uses a version of `@swc-node/register` thaqt does not match semver `^1.4.2` the `@nrwl/linter` packages uses the version listed in its dependencies. Npm or Yarn would create the following stucture: 

```
node_modules/
     ...
    @swc/register@another version
    ....
    @nrwl/linter
        node_modules
             @swc/register@^1.4.2
```

So Nx should not force users to use another version. I think a migration is only needed when `@swc-node/register` would be listed as `peerDependency`.

Reply:
You are correct! Thanks!